Resources/Prototypes/Entities/Objects/Specific/revolutionary.yml

```
@@ -22,6 +22,38 @@
       sound: /Audio/SimpleStation14/Items/Handling/book_drop.ogg
     - type: EmitSoundOnLand
       sound: /Audio/SimpleStation14/Items/Handling/book_drop.ogg
+    - type: ExtendDescription
+      descriptionList:
+      - description: "revolutionarymanifesto-extenddescription-headrevolutionary"
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)